presto-native-execution/presto_cpp/main/types/tests/PrestoToVeloxQueryPlanTest.cpp

```diff
@@ -0,0 +1,138 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "presto_cpp/main/types/PrestoToVeloxQueryPlan.h"
+#include <gtest/gtest.h>
+#include "velox/common/base/tests/GTestUtils.h"
+
+using namespace facebook::presto;
+using namespace facebook::velox;
+
+class PrestoToVeloxQueryPlanTest : public ::testing::Test {};
+
+TEST_F(PrestoToVeloxQueryPlanTest, parseSqlFunctionHandleWithZeroParam) {
+  std::string str = R"(
```

Comment:
So the error was very generic. 
Whenever we tried to parse any SqlFunctionHandle with zero / multiple params it lead to a never-ending loop, for eg: "functionId": "json.x4.sum;" failed because of zero params, and "functionId": "json.x4.sum;bigint;bigint;" will fail too.